SimCalorimetry/HcalSimProducers/src/HcalDigitizer.cc

```diff
@@ -83,6 +83,7 @@ namespace HcalDigitizerImpl {
 
 HcalDigitizer::HcalDigitizer(const edm::ParameterSet& ps, edm::ConsumesCollector& iC) :
   theGeometry(0),
+  theRecNumber(0),
```

Comment:
Thanks Yana - I tried to take care of those (and a few more)
________________________________
From: Ianna Osborne [notifications@github.com]
Sent: 15 August 2015 11:30
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-hcx25 Enable runing GEN-SIM-DIGI step with modified Hcal geometry definition (#10353)


In SimCalorimetry/HcalSimProducers/src/HcalDigitizer.cc<https://github.com/cms-sw/cmssw/pull/10353#discussion_r37134903>:

> @@ -83,6 +83,7 @@ namespace HcalDigitizerImpl {
>
>  HcalDigitizer::HcalDigitizer(const edm::ParameterSet& ps, edm::ConsumesCollector& iC) :
>    theGeometry(0),
> +  theRecNumber(0),


@bsunanda<https://github.com/bsunanda> :

SimCalorimetry/HcalSimProducers/src/HcalDigitizer.cc:86:3: error: class 'HcalDigitizer' does not have any field named 'theRecNumber'
   theRecNumber(0),
   ^



—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/10353/files#r37134903>.